modules/red-hat-marketplace-features.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * applications/red-hat-marketplace.adoc
+
+[id="red-hat-marketplace-features_{context}"]
+= Red Hat Marketplace features
+
+Cluster administrators can use link:https://marketplace.redhat.com/en-us/documentation/getting-started[Red Hat Marketplace] to manage software on {product-title}, give developers self-service access to deploy application instances, and correlate application usage against a quota.
+
+[id="marketplace-clusters_{context}"]
+== Connect {product-title} clusters to the Marketplace
+
+You can install a common set of applications on {product-title} clusters that you connect to the Marketplace. You can also use the Marketplace to track cluster usage against your subscriptions or quota. Additional users can be added
+// (NOTE: added to what?) 
```

Comment:
to the account to view Marketplace apps purchased, installed, etc

Reply:
@robszumski Do you think calling this out in the OCP docs is important, or could it be removed?

Reply:
it's a main use-case that helps you understand the product, so I would say yet it should stay

Reply:
Sure. Wasn't positive if user management was an 'expected to be there' sort of functionality or not. @robszumski